iidm/iidm-extensions/src/test/java/com/powsybl/iidm/network/extensions/VoltagePerReactivePowerControlXmlSerializerTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.AbstractConverterTest;
+import com.powsybl.iidm.export.Exporters;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.StaticVarCompensator;
+import com.powsybl.iidm.network.test.SvcTestCaseFactory;
+import com.powsybl.iidm.xml.NetworkXml;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Anne Tilloy <anne.tilloy at rte-france.com>
+ */
+public class VoltagePerReactivePowerControlXmlSerializerTest extends AbstractConverterTest {
+
+    @Test
+    public void test() throws IOException {
+        Network network = SvcTestCaseFactory.create();
+        network.setCaseDate(DateTime.parse("2019-05-27T12:17:02.504+02:00"));
+        StaticVarCompensator svc = network.getStaticVarCompensator("SVC2");
+        assertNotNull(svc);
+
+        svc.newExtension(VoltagePerReactivePowerControlAdder.class).withSlope(0.5).add();
+
+        // FIXME
```

Comment:
@mathbagu Help, it is not working and I think it is just because I am not familiar with xml conversion. Thanks !

Reply:
The error I get is:
````com.powsybl.commons.exceptions.UncheckedSaxException: org.xml.sax.SAXParseException; lineNumber: 24; columnNumber: 60; cvc-complex-type.2.4.c : Le caractère générique concordant est strict, mais aucune déclaration ne peut être trouvée pour l'élément 'vprpc:voltagePerReactivePowerControl'.````

Reply:
The problem was that namespace was different in xsd and in the XML serializer class.
I removed the FIXME annotation. 
I also fixed the expected value in the final assert (should be 0.5, that is the value used when creating the extension).

Reply:
Thanks a lot @zamarrenolm ! Have you check the cgmes conversion by the way ?

Reply:
Yes, I saw that you are storing in the extension the `slope` attribute read in the `staticVarCompensators` SPARQL query, so for me it is ok.